main.go

```diff
@@ -44,25 +44,27 @@ func main() {
 
 	r := chi.NewRouter()
 
+	var err error
```

Comment:
https://github.com/UCCNetsoc/fail2rest/pull/7/files?file-filters%5B%5D=.go&file-filters%5B%5D=.sum&file-filters%5B%5D=No+extension&w=1#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R57 needs to become `err = consul.Setup()` for this to be effective

Reply:
Oops, my mistake. Fixed.